docs/conf.py

```diff
@@ -196,7 +196,7 @@ def __getattr__(cls, name):
 # Example configuration for intersphinx: refer to the Python standard library.
 intersphinx_mapping = {
     'https://docs.python.org/3/': None,
-    'https://tskit.readthedocs.io/en/stable/': None,
```

Comment:
Need to temporarily change to the latest tskit docs to have the "trees_and_tree_sequences" section link.